plugins/quickinspector/quickinspector.cpp

```diff
@@ -243,9 +243,9 @@ static QString qsgTextureWrapModeToString(QSGTexture::WrapMode wrapMode)
 
 static bool isGoodCandidateItem(QQuickItem *item)
 {
-    if (!item->isVisible() || item->z() < 0 || qFuzzyCompare(item->opacity() + 1.0, qreal(1.0)) ||
```

Comment:
Yep, it's ugly code - I completely agree. But GammaRay tends to be used in the face of ugly/broken code and needs to cope with that.